sdk/identity/identity/src/credentials/vscodeCredential.ts

```diff
@@ -19,12 +19,27 @@ const VSCodeUserName = 'VS Code Azure';
 export class VSCodeCredential implements TokenCredential {
   private identityClient: IdentityClient;
 
+  /**
+   * Creates an instance of VSCodeCredential to use for automatically authenicating via VSCode.
+   * 
+   * @param options Options for configuring the client which makes the authentication request. 
+   */
   constructor(
     options?: TokenCredentialOptions
   ) {
     this.identityClient = new IdentityClient(options);
   }
 
+  /**
+   * Returns the first access token returned by one of the chained
+   * `TokenCredential` implementations.  Throws an {@link AggregateAuthenticationError}
+   * when one or more credentials throws an {@link AuthenticationError} and
+   * no credentials have returned an access token.
```

Comment:
looks like this was copied from `ChainedTokenCredential` and not updated

Reply:
Good catch. Fixed.